.github/workflows/release.yml

```diff
@@ -49,8 +49,8 @@ jobs:
       - name: Build packages
         run: yarn build
 
-      # - name: Build CRA templates
-      #   run: yarn build:templates ../cra-templates
+      - name: Build prop docs
+        run: yarn build:prop-docs
```

Comment:
What do you think about adding this to a `postbuild` script in the root package.json?

Reply:
I'm trying to reduce the frequency of running this script due to its slowness. Adding it to the `postbuild` script will make it run for every workflow action (PR and Release) when we only need this in the release workflow.